packages/requisitions/src/CustomerRequisition/ListView/ListView.tsx

```diff
@@ -47,7 +47,7 @@ export const CustomerRequisitionListView: FC = () => {
 
   const columns = useColumns<RequisitionRow>(
     [
-      { ...getNameAndColorColumn(), setter: onUpdate },
+      [getNameAndColorColumn(), { setter: onUpdate }],
```

Comment:
currently the requisitions ( customer & supplier ) aren't saving the change when updated using the sidepanel, though the comment is.

Reply:
ugh! Lucky you're here